.travis.yml

```diff
@@ -38,4 +45,4 @@ install:
 - scripts/install_test_dependencies.sh
 
 script:
-- scripts/run_test_suite.sh
+- scripts/check_versions.sh
```

Comment:
Ok, so we'll start using it again just for the version checks? Looks like it's a new beta feature of Travis that Circle doesn't have.